nle/scripts/ttyplay.py

```diff
@@ -126,6 +135,12 @@ def process(f):
             continue
 
         if channel == 1:  # Input channel.
+            os.write(
+                1, b"\033[s\033[26;0f\033[37;1mFrame %d:\033[0m " % frame
+            )  # Save Cursor & Jump to L26
```

Comment:
could this interfere with other saved cursors from the data stream?

Reply:
A quick google suggests, yes you can only save one cursor at a time, so yes this would interfere. On test rendering i've not seen any problems though for our ttyrec2s...